chain/block.go

```diff
@@ -117,7 +118,7 @@ type StatelessBlock struct {
 	vdrState     validators.State
 
 	results    []*Result
-	feeManager *FeeManager
+	feeManager *fees.FeeManager
```

Comment:
When we move this to `fees`, we should rename `FeeManager` -> `Manager` to avoid a stutter.